packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -597,6 +601,13 @@ function isValidKeyboardEvent(event: KeyboardEvent): boolean {
   );
 }
 
+function isSubmitEvent(event: KeyboardEvent): boolean {
+  const {key, target} = event;
+  const element = target as HTMLElement;
+  const type = element.getAttribute('type');
+  return (key === 'Enter' || key === ' ' || key === 'Spacebar') && type === 'submit';
```

Comment:
Not sure if we wanna handle this here, but perhaps we could do
```suggestion
  return (key === 'Enter' || key === ' ' || key === 'Spacebar') && (type === 'submit' || type === 'reset');
```
for reset handling? The function would then get renamed appropriately of course

Reply:
We'll add reset later when we also add onReset

Reply:
We already check `isValidKeyboardEvent` before this, so I don't think the key checking is necessary.